README.md

```diff
@@ -66,7 +71,12 @@ val producerConfig = {
 val dStream: DStream[String] = ...
 dStream.writeToKafka(
   producerConfig,
-  s => new ProducerRecord[String, String](topic, s)
+  s => new ProducerRecord[String, String](topic, s),
+  Some(new Callback with Serializable {
+    override def onCompletion(metadata: RecordMetadata, e: Exception) = {
+      if(Option(e).isDefined) "write failed!" else "write succeeded!"
+    }
+  })
```

Comment:
Sorry to be a pain, but I'd rather keep those examples as is and create another subsection named Callback with a separate example where you'd explain that we can also supply a callback.